generators/python/src/fern_python/generators/fastapi/core_utilities/core_utilities.py

```diff
@@ -134,6 +134,15 @@ def copy_to_project(self, *, project: Project) -> None:
             ),
             exports={"serialize_datetime"},
         )
+        self._copy_file_to_project(
```

Comment:
We should only copy this if relevant (not using string enum), check other Python generators for ref